module/IlluminaBeadArrayFiles.py

```diff
@@ -576,8 +576,14 @@ def normalize_intensities(self, x, y, threshold = True):
         tempx3 = tempx2 - self.shear * tempy2
         tempy3 = tempy2
 
-        xn = tempx3 / self.scale_x
-        yn = tempy3 / self.scale_y
+        if self.scale_x == 0:
+            xn = 0.0
```

Comment:
@ekarlins,
  Are you able to share an IDAT file and manifest that triggered this behavior? I suspect that it might make more sense to  set the normalized intensity to NaN here, but it would be useful to consider the appropriate behavior in the context of the full set of raw intensities and the related normalization transform calculated by AutoConvert.
  Thanks,
    Ryan